site2/docs/reference-configuration-broker.md

```diff
@@ -2329,6 +2329,26 @@ Interval between checks to see if message publish buffer size is exceed the max
 
 **Category**: Server
 
+### enableBrokerSideSubscriptionPatternEvaluation
```

Comment:
Hi @SignorMercurio 

before https://github.com/apache/pulsar/pull/16865 gets merged and takes effect, do we allow users to update configuration md files manually? I think no, right? 

Because after your PR gets merged and takes effect, the new changes will be synced and shown on the website, right？

Reply:
Totally right.

Reply:
@andrasbeni thanks for your contribution! We're trying to generate configuration doc from code automatically to avoid manual work https://github.com/apache/pulsar/issues/13453, could you please revert the changes to `.md` files?

Reply:
Thanks for the review. Changes to the .md files have been reverted.

Reply:
@Anonymitaet, as I can see, #16865 is not going to be merged. What can I do now to get this PR merged?

Reply:
@andrasbeni we can merge this PR after all tests pass